### DISCUSSION TITLE
Multiuser interface

### DISCUSSION THREAD

**Discussion by meszolym** [VueTorrent/VueTorrent#1084] [Ideas]
Hey,

My idea is to have a multiuser interface. Nothing too fancy, just the ability to add users with their own usernames + passwords, and being able to assign certain simple roles (admin, advanced user, downloader, etc).
Why this comes up: I have a torrentbox, me and couple of family members + friends use it. My problem is that all of them can see and change settings, which is not ideal.

This would be a really nice thing, I think!

**Comment by Larsluph** [comment]
> My idea is to have a multiuser interface.

Hey! Good idea!

Unfortunately you're not at the right place. VueTorrent is only a WebUI for qBittorrent, so we're unable to implement such feature. I would suggest submiting your idea to the official qBittorrent repo 😉
https://github.com/qbittorrent/qBittorrent

**Reply by meszolym** [reply]
I know, but I think that this can be carried out by the WebUI actually, we're not talking about anything too fancy here in my opinion. Anyways, I will also submit this there, but as I heard they are not planning on including this feature.

**Reply by Larsluph** [reply]
No that's literally impossible for a WebUI.

We don't have anything to do with qBittorrent, we only use the API to fetch torrent data.

User management is 100% done in qBittorrent and we can't interfere with this as it is needed to authenticate API calls to fetch anything coming from the API.

**Reply by WDaan** [reply]
As @Larsluph is saying, there are 2 issues with this:
- we can't store anything server side, all data remains in the browser. So if you create a user on your end there is literally no way to show that user on someone else's end
- we still need to be authenticated against the api itself to read info about torrents
